docs/Unicode.md

```diff
@@ -140,17 +176,11 @@ pdf.output("unicode.pdf")
 View the result here: 
 [unicode.pdf](https://github.com/py-pdf/fpdf2/raw/master/tutorial/unicode.pdf)
 
```

Comment:
If all the text if right-to-left and text shaping is enabled there is no need for this workaround. However if there is any bidirectional segment (RTL inside a LTR text or the other way around) fpdf will render in a single direction for now.

Reply:
So the workaround is only needed for mixed direction text?

We also currently still have #901, which we should probably mention in this context.

Reply:
#901 is also being fixed on the bidirectional PR.
I guess we can leave this section as-is and I change this document on my PR.